pack/services/osp_services.cfg

```diff
@@ -0,0 +1,28 @@
+define service{
+   service_description          VMs-Count
+   use            osp-service
+   register       0
+   host_name	  osp
+   check_command  check_vm_count
+}
+define service{
+   service_description          Keypair-Count
+   use            osp-service
+   register       0
+   host_name	  osp
+   check_command  check_keypair_count
+}
+define service{
+   service_description          Volume-Count
+   use            rhv-service
+   register       0
+   host_name	  osp
+   check_command  check_volume_count
+}
+define service{
+   service_description          Hosts-Services-Status
+   use            osp-service
+   register       0
+   host_name     osp
+   check_command  check_services_running
```

Comment:
This is call `check_host_services_running` in the command you've defined above. I would call it something unique to distinguish it from the rhv-check.